.github/workflows/FastTests.yml

```diff
@@ -0,0 +1,36 @@
+name: FastTests
+
+on:
+  push:
+    branches:
+        - v5.0
+        - v4.2
+        - zstd
```

Comment:
Should be removed?

Reply:
Yes, I think we might want to get that file to 4.2 branch as well right?